protocol/communities_messenger_test.go

```diff
@@ -3256,6 +3256,89 @@ func (s *MessengerCommunitiesSuite) TestSetMutePropertyOnChatsByCategory() {
 	}
 }
 
+func (s *MessengerCommunitiesSuite) TestMuteAllCommunityChats() {
+	// Create a community
+	createCommunityReq := &requests.CreateCommunity{
+		Membership:  protobuf.CommunityPermissions_ON_REQUEST,
+		Name:        "new community",
+		Color:       "#000000",
+		Description: "new community description",
+	}
+
+	mr, err := s.alice.CreateCommunity(createCommunityReq, true)
+	s.Require().NoError(err, "s.alice.CreateCommunity")
+	var newCommunity *communities.Community
+	for _, com := range mr.Communities() {
+		if com.Name() == createCommunityReq.Name {
+			newCommunity = com
+		}
+	}
+	s.Require().NotNil(newCommunity)
+
+	orgChat1 := &protobuf.CommunityChat{
+		Permissions: &protobuf.CommunityPermissions{
+			Access: protobuf.CommunityPermissions_NO_MEMBERSHIP,
+		},
+		Identity: &protobuf.ChatIdentity{
+			DisplayName: "status-core",
+			Emoji:       "😎",
+			Description: "status-core community chat",
+		},
+	}
+
+	orgChat2 := &protobuf.CommunityChat{
+		Permissions: &protobuf.CommunityPermissions{
+			Access: protobuf.CommunityPermissions_NO_MEMBERSHIP,
+		},
+		Identity: &protobuf.ChatIdentity{
+			DisplayName: "status-core2",
+			Emoji:       "😎",
+			Description: "status-core community chat2",
+		},
+	}
+
+	mr, err = s.alice.CreateCommunityChat(newCommunity.ID(), orgChat1)
+	s.Require().NoError(err)
+	s.Require().NotNil(mr)
+	s.Require().Len(mr.Communities(), 1)
+	s.Require().Len(mr.Chats(), 1)
+
+	mr, err = s.alice.CreateCommunityChat(newCommunity.ID(), orgChat2)
+	s.Require().NoError(err)
+	s.Require().NotNil(mr)
+	s.Require().Len(mr.Communities(), 1)
+	s.Require().Len(mr.Chats(), 1)
+
+	muteDuration, _ := s.alice.MuteDuration(MuteFor15Min)
+
+	_, err = s.alice.MuteAllCommunityChats(newCommunity.IDString(), true, MuteFor15Min)
+	s.Require().NoError(err)
+
+	aliceCommunity, err := s.alice.GetCommunityByID(newCommunity.ID())
+	s.Require().NoError(err)
+	s.Require().True(aliceCommunity.Muted())
+
+	for _, chat := range s.alice.Chats() {
+		if chat.CommunityID == newCommunity.IDString() {
+			s.Require().True(chat.Muted)
+			s.Require().Equal(chat.MuteTill, muteDuration)
+		}
+	}
+
+	chat := s.alice.Chats()[0]
+
+	err = s.alice.UnmuteChat(chat.ID)
+	s.Require().NoError(err)
+	s.Require().False(chat.Muted)
+
+	_, err = s.alice.MuteAllCommunityChats(newCommunity.IDString(), false, MuteFor15Min)
```

Comment:
I prefer to have either two functions `MuteAllCommunityChats` and `UnmuteAllCommunityChats`, or at least rename the function so it's clear that is not just muting, something like `SetMutedOnAllCommunityChats` or similar